cmd/access/main.go

```diff
@@ -89,6 +90,8 @@ func main() {
 			flags.StringVarP(&rpcConf.HistoricalAccessAddrs, "historical-access-addr", "", "", "comma separated rpc addresses for historical access nodes")
 			flags.DurationVar(&rpcConf.CollectionClientTimeout, "collection-client-timeout", 3*time.Second, "grpc client timeout for a collection node")
 			flags.DurationVar(&rpcConf.ExecutionClientTimeout, "execution-client-timeout", 3*time.Second, "grpc client timeout for an execution node")
+			preferredExecutionNodeIDs := common.IdentifierListValue(rpcConf.PreferredExecutionNodeIDs)
+			flags.Var(&preferredExecutionNodeIDs, "nodeids", "usage")
```

Comment:
`flags.Var` to use the custom command line argument parser

Reply:
nodeids seems to be a general name, maybe "preferred-execution-nodes"

Reply:
you might also update the "usage" with more data and examples